python-api/app/integrations/openrouter/client.py

```
@@ -1,180 +1,440 @@
-import os
-import json
+"""
+Main OpenRouter client implementation with async support.
+"""
 import logging
-from fastapi import HTTPException
-import requests
-from dotenv import load_dotenv
-from typing import List, Generator, Union
+from typing import List, Dict, Any, Optional, AsyncGenerator, Union, Generator
```

Ruff (0.12.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)